t/test/mojo.t

```diff
@@ -217,4 +221,45 @@ subtest 'header_unlike' => sub {
   is_deeply \@args, ['unlike', 'text/html;charset=UTF-8', qr/image\/png/, 'some description'], 'right result';
 };
 
+subtest 'message_is' => sub {
+  $t->websocket_ok('/websocket')->message_ok->message_is('testing message');
+  is_deeply \@args, ['is', ('testing message') x 2, 'exact match for message'], 'message_is succeded.';
```

Comment:
Don't get fancy with the `x 2`, we prefer readability in tests, even if it ends up a little more verbose.

Reply:
And please use test descriptions consistent with other tests.

Reply:
Done